aws/github-org-ecr/main.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = var.aws_assume_role_arn
+  }
+}
+
+provider "github" {
+  token        = var.github_token
```

Comment:
@maximmi 
Is that module to create ECR for all repos, including private, or just public repos?
If public, then we don't need to provide GH token, we can implement what was done here https://github.com/cloudposse/terraform-github-repository-webhooks/pull/18/files#diff-7a370d8342e7203b805911c92454f0f4R4

Reply:
it's predominantly for *private* images ;)